src/views/virtualmachinesinstance/list/utils/virtualMachinesInstancesRowFilter.ts

```diff
@@ -0,0 +1,51 @@
+import { V1VirtualMachineInstance } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { FilterValue, RowFilter } from '@openshift-console/dynamic-plugin-sdk';
+
+import { VM_KUBEVIRT_OS_ANNOTATION } from './virtualMachinesInstancesConstants';
+import { osNames, vmiStatuses } from './virtualMachinesInstancesStatuses';
+
+const osTitles = {
+  centos: 'Centos',
+  fedora: 'Fedora',
+  windows: 'Windows',
+  opensuse: 'OpenSuse',
```

Comment:
no need for opensuse or Ubuntu filter.
create a `Other` label for them.

Reply:
done